src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -121,27 +113,140 @@ public SyntaxNode Mutate(SyntaxNode currentNode)
+                    forStatement = MutateInPlace(originalFort, originalFort.Condition);
+                    forStatement = MutateInPlace(forStatement, forStatement.Statement);
+                    return newNode.ReplaceNode(originalFort, forStatement);
+                }
+                else
+                {
+                    result = MutateWithIfStatements(statement);
+                }
+
+                if (result != statement)
+                {
+                    var node = ((result as IfStatementSyntax).Else.Statement as BlockSyntax).Statements[0];
+                    return result.ReplaceNode(node, MutateExpressions(node));
+                }
+            }
+
+            var replaceNode = MutateExpressions(currentNode);
+            if (replaceNode != null)
+            {
+                return replaceNode;
+            }
+
+            // No statement found yet, search deeper in the tree for statements to mutate
+            var children = currentNode.ChildNodes().ToList();
+            var childCopy = currentNode.TrackNodes(children);
+            foreach (var child in children)
+            {
+                var originalNode = childCopy.GetCurrentNode(child);
+                var mutatedNode = Mutate(child);
+                if (!mutatedNode.IsEquivalentTo(originalNode))
+                {
+                    childCopy = childCopy.ReplaceNode(originalNode, mutatedNode);
+                }
             }
-            else
+            return childCopy;
+        }
+
+        private SyntaxNode MutateExpressions(SyntaxNode currentNode)
+        {
+            var expressions = GetExpressionSyntax(currentNode).Where(x => x != null);
+            if (expressions.Any())
             {
-                // No statement found yet, search deeper in the tree for statements to mutate
-                var children = currentNode.ChildNodes().ToList();
-                var childCopy = currentNode.TrackNodes(children);
-                foreach (var child in children)
+                var currentNodeCopy = currentNode.TrackNodes(expressions);
+                foreach (var expressionSyntax in expressions)
                 {
-                    var mutatedNode = Mutate(child);
-                    var originalNode = childCopy.GetCurrentNode(child);
-                    if (!mutatedNode.IsEquivalentTo(originalNode))
+                    var currentExpressionSyntax = currentNodeCopy.GetCurrentNode(expressionSyntax);
+                    SyntaxNode actualMutationNode = null;
+                    if (currentNode is AssignmentExpressionSyntax assign)
+                    {
+                        actualMutationNode = assign.Right;
+                    }
+                    else if (currentNode is ExpressionStatementSyntax)
+                    {
+                        // The expression of a ExpressionStatement cannot be mutated directly
+                        return MutateInPlace(currentNodeCopy, currentExpressionSyntax);
```

Comment:
Mutate should not be called with a copy of the current node. This is where the parent node is already `null`. All mutations produced by this statement will have incorrect spans (and thus incorrect locations in the report).

Reply:
Thanks for pointing my errors.
Fixed